config/secrets.yml

```diff
@@ -15,8 +15,19 @@ development:
 
 test:
   secret_key_base: a4601e0fb037b7d1ed534a5d5dd114f47b753c9945757ebe0f9fedff98d41b12741f7dd631e0d9a69cd518623384d871b0387ec5645ccccbbb5d9336ff1cf800
+  sfax_username: sfax_username
+  sfax_api_key: sfax_api_key
+  sfax_encryption_key: sfax_encryption_key
+  sfax_init_vector: sfax_init_vector
+
+from_environment: &deployable_settings
+  sfax_username: <%= ENV['SFAX_USERNAME'] %>
```

Comment:
I wonder - is there any downside to just referencing `ENV['WHATEVER']` in the code itself? We have other keys which we are managing directly from the environment variables. ([here's an example of the SMTP credentials][1])

[1]: https://github.com/codeforamerica/michigan-benefits/blob/3ece19a001d1ea57cf4d228067cc2841e279a397/config/smtp.rb